<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="wordsListTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wordLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="noteEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="addEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="addNoteButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Add Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWordButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Add Word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knowButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Known</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forgetButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Forget</string>
    </property>
   </widget>
   <widget class="QTableView" name="linkTableView">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linkButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Link Word</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="seen">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Seen:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nSeenLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="forget">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Forget:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nForgetLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="importance">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Importance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nImpoLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="total">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="thisrun">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>This run:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="forget_2">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Forget:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="nTotalLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nRunLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nRunForgetLabel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWordLink">
    <property name="title">
     <string>A&amp;nalysis</string>
    </property>
    <addaction name="actionDate_path"/>
    <addaction name="actionMemory_Curve"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionMannual"/>
   </widget>
   <addaction name="menuWordLink"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDate_path">
   <property name="text">
    <string>Date Path</string>
   </property>
  </action>
  <action name="actionMannual">
   <property name="text">
    <string>Mannual</string>
   </property>
  </action>
  <action name="actionMemory_Curve">
   <property name="text">
    <string>Memory Curve</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
